README.md

```diff
@@ -1,2 +1,71 @@
-# content-providers
-A collection of content providers for accessing notebooks in different storage locations
+#@nteract/google-content-provider
+## Installation
+Installation
+To use google cloud storage, you can use the @nteract/google-content-provider package.
+```
+$ npm install --save @nteract/google-content-provider
+
+```
+## Usage
+## <a name="create-bucket"></a> Create your Bucket on Google Cloud Storage
+The bucket should be created with **fine grained** access control, as the plugin will configure uploaded files with public read access.
+### How to create a bucket ?
+- https://cloud.google.com/storage/docs/creating-buckets
+
+### Where my bucket can be located ?
+- https://cloud.google.com/storage/docs/locations
+
+## Setting up Google authentication
```

Comment:
We don't need to include explicit instructions for this. We can refer users to the Google Cloud docs. This way, if Google changes anything we don't have to worry about updating our README.

Instead, focus on how to use the API here.